particles/Tutorial/JsonStore/JsonStore.recipe

```diff
@@ -0,0 +1,27 @@
+// Tutorial 3: JSON Store
+// Loads data stored in a JSON file.
+
+// Defines a new Entity called PersonDetails, with two fields.
+schema PersonDetails
+  Text name
+  Number age
+
+// Creates a data store of type PersonDetails, named PersonToGreetStore (the name is not important, it isn't used anywhere). The data will be
+// loaded from the file data.json. This store is tagged with #personToGreet, which lets us reference it in recipes.
+store PersonToGreetStore of PersonDetails #personToGreet in 'data.json'
```

Comment:
Some fyi's...

You can actually use the name to reference the store:
`map PersonToGreetStore as data`

And you can give them an id which is also usable:
`store PersonToGreetStore of PersonDetails 'personToGreet' in 'data.json'`
`map 'personToGreet' as data`

Also, json data can be stored directly in a manifest:
```
store PersonToGreetStore of PersonDetails 'personToGreet' in PersonData

resource PersonData
  start
  [
    {"name": "Jill", "age": 70}
  ]
```

Reply:
That is really helpful to know, thanks!

Do you have any tips on when to which? `#personToGreet` vs. `'personToGreet'` vs `PersonToGreetStore`?

I might add some more examples after showing the different ways of connecting things.